tabular/src/autogluon/tabular/models/automm/ft_transformer.py

```diff
@@ -98,3 +98,7 @@ def _get_default_ag_args_ensemble(cls, **kwargs) -> dict:
         }
         default_ag_args_ensemble.update(extra_ag_args_ensemble)
         return default_ag_args_ensemble
+
+    @classmethod
+    def _class_tags(cls):
+        return {'handles_text': False}
```

Comment:
Maybe move these constants to a separate constant class where they can be fully documented + reduces likelihood of typos.

Reply:
scikit-learn just uses strings: https://github.com/scikit-learn/scikit-learn/blob/ac665ad80be586a90c7d353ed37f1e64a4f547a1/sklearn/neighbors/_classification.py#L712-L713

I think we can also just use strings unless it becomes a problem.